paddle/fluid/operators/top_k_op.h

```diff
@@ -55,6 +55,9 @@ class TopkKernel : public framework::OpKernel<T> {
     // NOTE: eigen shape doesn't affect paddle tensor.
     eg_input.reshape(flat2dims);
 
+#ifdef PADDLE_WITH_MKLDNN
```

Comment:
how about use `PADDLE_WITH_MKLML`, since the speedup of `top_k_op` can be used in MKL scenario.